code/game/objects/items/weapons/stunbaton.dm

```diff
@@ -132,13 +101,15 @@
 	add_fingerprint(user)
 
 
-/obj/item/weapon/baton/attack(mob/M, mob/user)
-	if(has_user_lock && !skillcheck(user, SKILL_POLICE, SKILL_POLICE_SKILLED))
-		to_chat(user, SPAN_WARNING("You don't seem to know how to use [src]..."))
-		return
-
-	var/stun = stunforce
-	var/mob/living/L = M
+/obj/item/weapon/baton/attack(mob/target, mob/user)
+	var/mob/living/carbon/human/real_user = user
+	var/mob/living/carbon/human/human_target = target
+	if(has_user_lock && !skillcheck(real_user, SKILL_POLICE, SKILL_POLICE_SKILLED) && status)
```

Comment:
Remove the status check here. It effectively makes this whole ordeal pointless because these batons lose charge very quickly. If you want an IC reason; it has residual charge.